### PR TITLE
Add a footer to the website that credits CGDS

### PR DESCRIPTION
**Pull request by kuthedk**
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines enforced by static analysis tools.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.

## Pull Request Details

Wrike Ticket - [On the frontned, add a tiny footer that credits CGDS developing Rosalution](https://www.wrike.com/open.htm?id=1033379957)

Changes made:

New component AppFooter.vue
- Created a new vue component named AppFooter

App.vue
- Added a new component, `AppFooter` inside the template and enclosed router-view inside a div 

main.js
- A new component `AppFooter` was imported from ./components/AppFooter.vue and registered as a component.
- The component `AppFooter` is added as a global component.

**To Review:**

- [x] Static Analysis by Reviewer
- [x] Clicking on the footer takes you to https://github.com/uab-cgds-worthey/rosalution
- [x] All Github Actions checks have passed.

### Screenshots
image
image

### CODE REVIEW

File: frontend/src/components/AppFooter.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <footer>
+    <a class="footer-link" href="https://github.com/uab-cgds-worthey/rosalution/blob/main/COPYING">
+      Copyleft <span class="copy-left">&copy;</span> {{ currentYear }} Center for Computational Genomics and Data
```
Comment (SeriousHorncat) [inline frontend/src/components/AppFooter.vue:4]: Hmmm, we didn't discuss including the copyleft license in the footer.  This is really interesting.  Have you seen this done somewhere before?  I was just imaging the name of the center, and the fully qualified name of the schoool of medicine.

File: frontend/src/components/AppFooter.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      Science, UAB School of Medicine
```
Comment (SeriousHorncat) [inline frontend/src/components/AppFooter.vue:5]:
```suggestion
      Science, University of Alabama at Birmingham, Heersink School of Medicine
```

**Review by SeriousHorncat — COMMENTED** [review]
Lets chat tomorrow on the specific verbiage that can go in here.  I can get caught up with things going on.

File: frontend/src/components/AppFooter.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--primary-background-color);
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
```
Comment (fatimarabab) [inline frontend/src/components/AppFooter.vue:31]:
```suggestion
  width: 100%;
  padding-bottom: var(--p-1);
```

File: frontend/src/components/AppFooter.vue [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+.footer-link {
+  font-size: 0.75em;
+}
+
+.copy-left {
+  display: inline-block;
+  text-align: right;
+  margin: 0px;
+  transform: scaleX(-1);
+}
```
Comment (fatimarabab) [inline frontend/src/components/AppFooter.vue:43]:
```suggestion
```

**Comment by fatimarabab** [comment]
Added a couple of comments to add bottom padding to footer and remove extra unused css class. Otherwise looks good and works as expected.

**Review by fatimarabab — CHANGES_REQUESTED** [review]

**Review by SeriousHorncat — APPROVED** [review]

**Review by fatimarabab — APPROVED** [review]

**Pull request merged**